README.md

```diff
@@ -16,12 +16,12 @@ One crucial thing to know before starting is the docker-pi-hole container needs
 
 ```
 IMAGE='diginc/pi-hole'
-NIC='eth0'
-IP=$(ip addr show $NIC | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
+IP_LOOKUP="$(ip route get 8.8.8.8 | awk '{ print $NF; exit }')"  # May not work for VPN / tun0
```

Comment:
this is pretty cool, but it doesn't work on modern versions of iproute2 the last parameters on the line are `uid <uid number>`

Reply:
Modern as in 4.X?
Could you provide a sample output from `ip route get 8.8.8.8`?

Reply:
```
8.8.8.8 via 192.168.0.1 dev wlp4s0 src 192.168.0.227 uid 1000 
    cache 
```

my iproute2 version is 4.15.0-2ubuntu1

Reply:
Thank you, I'll open a new issue to document this bug